CHANGELOG.md

```diff
@@ -5,6 +5,10 @@ This CHANGELOG follows the format listed [here](https://github.com/sensu-plugins
 
 ## [Unreleased]
 
+## [3.0.1] - 2018-01-21
+### Changed
```

Comment:
This should actually go under a `### Security` header rather than changed as it makes it clear to consumers that this update is important and should be prioritized over pulling in other types of changes.